### PR TITLE
(fix) updates content of confirmation page

### PR DESCRIPTION
**Pull request by gazaston**
# Do not merge! Awaiting URL for feedback page!

- takes pattern from [GOV UK design system](https://design-system.service.gov.uk/patterns/confirmation-pages/)
- removes some redundant specs
- adds a css utility for links against a coloured background

before:
before

after:
after

### CODE REVIEW

**Comment by gazaston** [comment]
[Update confirmation page contents](https://trello.com/c/cabv5h73/288-update-confirmation-page-contents)

File: app/views/hiring_staff/vacancies/summary.html.haml [hunk 1]
```diff
@@ -1,21 +1,25 @@
-- content_for :page_title_prefix, "Your job posting for #{@school.name} has been submitted"
+- content_for :page_title_prefix, "The job listing for #{@school.name} has been completed"
```
Comment (despo) [inline app/views/hiring_staff/vacancies/summary.html.haml]: Can this text be extracted to the locale file please.

File: app/views/hiring_staff/vacancies/summary.html.haml [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
-      %h1.heading-large.mv0
+      %h1.heading-large.mt0
         = t('jobs.submitted')
 
+      - if @vacancy.publish_today?
+        %p= link_to t('jobs.preview_posted_job'), job_path(@vacancy), class: 'reversed-link'
+      - else
+        %p= link_to t('jobs.preview_posted_job'), school_job_path(@vacancy.id), class: 'reversed-link'
+
+      %p.mb0 You can also view, amend or delete it on #{link_to "your school's dashboard", new_sessions_path, class: 'reversed-link'}
```
Comment (despo) [inline app/views/hiring_staff/vacancies/summary.html.haml]: Can this be extracted to the locale file please.

File: app/views/hiring_staff/vacancies/summary.html.haml [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
-      %p
-        = t('jobs.view_posted_job')
-      %p
-        = link_to job_url(@vacancy), job_path(@vacancy)
-    - else
-      %p
-        = t('jobs.preview_posted_job', date: @vacancy.publish_on)
-      %p
-        = link_to school_job_url(@vacancy.id), school_job_path(@vacancy.id)
+    %h3.heading-medium What happens next
```
Comment (despo) [inline app/views/hiring_staff/vacancies/summary.html.haml]: Can this be extracted to the locale file please along with the rest of the text in this page.

**Pull request closed without merging**